### ISSUE TITLE
Option autocomplete does not work in nested attrsets

### ISSUE THREAD

**Issue by vkryachko** [nix-community/nixd#264]
This file can be used to reproduce [module.nix](https://github.com/nix-community/nixd/blob/main/docs/examples/options/nixos/module.nix)

Autocomplete works for this expression:
```nix
{
  boot.binfmt
}
```

Doesn't work for this expression:

```nix
{
  boot = {
    binf # <- nixd here thinks we are at the root attrpath, not nested inside boot, and autocompletes root level properties
  };
}
```

**Comment by inclyc** [comment]
Duplicate to #247 
Fixed in #249 

This patch has not been released yet, because there are many regression issues in the frontend (nix language parser).

The new frontend has a large diff and currently cannot be merged. #257

**Comment by vkryachko** [comment]
@inclyc Thanks for the reply. Just to clarify: I tried building nixd at head after #249 and still observing lack of autocompletion. Does it mean that this will be addressed by the new parser in #257 ? in which case does it make sense to keep this issue open? thanks

**Comment by inclyc** [comment]
> @inclyc Thanks for the reply. Just to clarify: I tried building nixd at head after #249 and still observing lack of autocompletion. Does it mean that this will be addressed by the new parser in #257 ? in which case does it make sense to keep this issue open? thanks

Ahh, let's reopen #249 then (this issue is duplicated to it).
